pkg/services/object/acl/v2/types.go

```diff
@@ -0,0 +1,29 @@
+package v2
+
+import (
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	cidSDK "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	eaclSDK "github.com/nspcc-dev/neofs-sdk-go/eacl"
+	"github.com/nspcc-dev/neofs-sdk-go/owner"
+)
+
+// ACLChecker is an interface that must provide
+// ACL related checks.
+type ACLChecker interface {
+	// BasicACLCheck must return true only if request
+	// passes basic ACL validation.
+	BasicACLCheck(RequestInfo) bool
```

Comment:
I'd start the names from verb 
```suggestion
	CheckBasicACL(RequestInfo) bool
```